### PR TITLE
Configuration file .packit.yaml

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This PR adds support for packit https://github.com/packit-service/packit, which syncs upstream changes into downstream automatically. See docu on packit side https://github.com/packit-service/packit/blob/master/README.md.

### CODE REVIEW

**Comment by jkonecny12** [comment]
Looks good to me. Thanks!

**Comment by dcantrell** [comment]
Some comments:

- Can the .packit.yml file just be named packit.yml?  I don't mind it being visible in the default ls(1) output and would prefer to see that this is a packit-capable project.

- The spec file should be a template.  Please rename to pykickstart.spec.in and apply the comments I added to the file in the PR.

**Comment by phracek** [comment]
I will create a separate PR with SPEC file soon. Please be patient.

File: packit.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - packit.yaml
+  - pykickstart.spec.in
+jobs:
+  - trigger: release
+    release_to:
+      - master
+  - trigger: pull_request
+    release_to:
+      - master
+upstream_name: pykickstart
```
Comment (TomasTomecek) [inline packit.yaml]:
```suggestion
upstream_project_name: pykickstart
```
we have decided to rename these two keys to be more descriptive, the change will land in packit 0.2 which we release this week

File: packit.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+package_name: pykickstart
```
Comment (TomasTomecek) [inline packit.yaml]:
```suggestion
downstream_package_name: pykickstart
```

**Comment by phracek** [comment]
This pull request adds only `packit.yaml` file. NOT SPEC file.

**Comment by phracek** [comment]
@dcantrell I have updated the pull request. It contains only `packit.yaml` file. \
The other PR https://github.com/dcantrell/pykickstart/pull/262 contains `pykickstart.spec.in` as a template file.

**Comment by dcantrell** [comment]
Looks good, thanks.  Awaiting changes on the spec PR review.

**Pull request merged**